riemann/tx/tx.py

```diff
@@ -442,10 +442,6 @@ def from_bytes(InputWitness, byte_string):
             items.append(item)
         return InputWitness(items)
 
-    def copy(self, stack=None):
```

Comment:
Where did this go?

Reply:
done R-U-N-N-O-F-T it appears, have sent the bloodhounds